src/main/java/com/alkemy/ong/domain/ListComments.java

```diff
@@ -0,0 +1,17 @@
+package com.alkemy.ong.domain;
+
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class ListComments {
```

Comment:
Este objeto no agrega valor al dominio a menos que incorpore algún otro atributo.

Reply:
@JackyOS BE CAREFUL! Agregando otro atributo a este objeto de domino... es una alarma de que ya existe algo en el dominio que representa a esto. Los objetos del dominio representan cosas del negocio... ListComments no es un objeto de negocio.